Libraries/Core/Timers/JSTimersExecution.js

```diff
@@ -45,6 +45,7 @@ const JSTimersExecution = {
   timerIDs: ([] : Array<?number>),
   immediates: [],
   requestIdleCallbacks: [],
+  requestIdleCallbackTimeouts: ([] : Array<?number>),
```

Comment:
I think it would be better to use a Map here with the requestIdleCallback timer id as the key and the timeout timer id as the value.